setup.cfg

```diff
@@ -39,6 +39,7 @@ python_requires = >=3.6
 install_requires =
     dataclasses; python_version<"3.7"
 tests_require =
+    click
```

Comment:
is this necessary? or is having it as a requirement in tox enough?

Reply:
No, I don't think this is needed. Shouldn't there be an `extras_require` though?

Reply:
yes. done.